controllers/link.go

```diff
@@ -146,6 +151,55 @@ func GetLinkWithPassword(c *gin.Context) {
 	c.JSON(http.StatusOK, link)
 }
 
+func UpdateLink(c *gin.Context) {
+	var input UpdateLinkInput
+	if err := c.ShouldBindJSON(&input); err != nil {
+		c.JSON(http.StatusUnprocessableEntity, err.Error())
+		return
+	}
+
+	if input.Password == "" && input.Alias == "" {
```

Comment:
how about add `binding:required` to `UpdateLinkInput` struct on two of these fields?
what do you think?

Reply:
Currently, we just want to required at least one field, can we do that with `binding:required`?

Reply:
got it